src/python/py_imagebuf.cpp

```diff
@@ -361,7 +361,9 @@ declare_imagebuf(py::module& m)
              "height"_a = 0, "depth"_a = 0)
         .def("spec", &ImageBuf::spec,
              py::return_value_policy::reference_internal)
-        .def("nativespec", &ImageBuf::nativespec,
+        .def("file_format", &ImageBuf::file_format,
```

Comment:
I think here, like with C++, for back compatibility we'll want to preserve a nativespec() call, even if it does something different than before.